src/ray/raylet/worker_pool.cc

```diff
@@ -314,7 +314,12 @@ Process WorkerPool::StartWorkerProcess(const Language &language,
   ProcessEnvironment env;
   if (RayConfig::instance().enable_multi_tenancy()) {
     env.insert(job_config->worker_env().begin(), job_config->worker_env().end());
+
+    for (const auto &pair : override_worker_env) {
+      env[pair.first] = pair.second;
+    }
```

Comment:
Same here. The for loop can be put out of the if statement.

Reply:
This is a good point.